MeetU/MeetU/Scripts/app/meetu-meetup.js

```diff
@@ -3,7 +3,7 @@
 //  used for template file : ~/Meetups/Index.cshtml.
 //
 (function () {
-
+    "use strict";
```

Comment:
you may want to add you review here in the diff page. It is way easier to trace the issue